ext/winevt/winevt_channel.c

```diff
@@ -253,9 +276,21 @@ rb_winevt_channel_each(VALUE self)
       rb_raise(rb_eRuntimeError, errBuf);
```

Comment:
`winevtChannel->channels` should be closed in this block too.

Reply:
Yeah, `winevtChannel->channels` should be closed here.

Reply:
Fixed in https://github.com/fluent-plugins-nursery/winevt_c/pull/23/commits/dc93ef083521bddf28dc2c9efa34643ec8988982.